dds/idl/be_produce.cpp

```diff
@@ -107,6 +108,22 @@ BE_abort()
 
 namespace {
 
+///Only warn once per run
+bool pragmaWarn = false;
+/**
+ * @brief prints a warning if a #pragma is detected based off of
+ * IDL_GlobalData::dcps_type_info_map_::cur_size_
+ * @author ceneblock
+ */
+void warnOnPragma()
+{
+  std::cerr << "WARNING: #pragma is discontinued\n" <<
```

Comment:
Change this from "discontinued" to "deprecated", since we haven't actually removed them yet.